### PR TITLE
chore: add tests & cleanup flexible/java-11 samples

### PR DESCRIPTION
**Pull request by kweinmeister**
* Add basic tests to `websocket-jetty`
* Fix hardcoded projectId in `websocket-jetty`
* Remove unnecessary dependencies
* Upgrade dependencies
* Lint markdown files
* Explicitly set `runtime_config` to 11 in `app.yaml`
* Addressed several warnings

### CODE REVIEW

File: flexible/java-11/static-files/pom.xml [hunk 1]
```diff
@@ -76,22 +75,17 @@
...
       <scope>test</scope>
       <version>4.12.3</version>
-    </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty.toolchain</groupId>
-      <artifactId>jetty-test-helper</artifactId>
-      <version>6.0</version>
-      <scope>test</scope>
-    </dependency>
+    </dependency> -->
```
Comment (Shabirmean) [inline flexible/java-11/static-files/pom.xml:78]: why commented and not removed here?

**Review by Shabirmean — APPROVED** [review]
LGTM! One minor review comment

**Review by Shabirmean — APPROVED** [review]
LGTM! One minor review comment

File: flexible/java-11/static-files/pom.xml (hunk 1 above)
Comment (kweinmeister) [reply flexible/java-11/static-files/pom.xml]: Good catch! Removed instead.

**Comment by kweinmeister** [comment]
> LGTM! One minor review comment

Thanks @Shabirmean

**Pull request merged**